tests/test/org/commcare/cases/CaseTestUtils.java

```diff
@@ -0,0 +1,58 @@
+package org.commcare.cases;
+
+import org.commcare.test.utils.TestInstanceInitializer;
+import org.commcare.util.mocks.MockUserDataSandbox;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.ExternalDataInstance;
+import org.javarosa.model.xform.DataModelSerializer;
+import org.javarosa.xpath.XPathParseTool;
+import org.javarosa.xpath.expr.XPathExpression;
+import org.javarosa.xpath.expr.XPathFuncExpr;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class CaseTestUtils {
+    public static final String CASE_INSTANCE = "jr://instance/casedb";
+    public static final String LEDGER_INSTANCE = "jr://instance/ledgerdb";
+
+    public static byte[] loadCaseInstanceIntoSandbox(MockUserDataSandbox sandbox) {
```

Comment:
This is a confusingly named method. Should be

serializeCaseInstanceFromSandbox

or something similar